include/psa/crypto_struct.h

```diff
@@ -68,6 +68,9 @@ extern "C" {
 #include "psa/crypto_driver_contexts_primitives.h"
 
```

Comment:
Looks good so far but please do all structures, not just the ones that TF-M happens to use at the moment. Missing:
```
struct psa_sign_hash_interruptible_operation_s
struct psa_verify_hash_interruptible_operation_s
```
and also in `crypto_extra.h`:
```
struct psa_pake_operation_s
```

Reply:
Done